assets/src/stories-editor/components/editor-carousel/edit.css

```diff
@@ -68,3 +68,55 @@
 .editor-styles-wrapper ul.amp-story-editor-carousel-item-list .amp-story-editor-carousel-item--active button {
 	background: #666;
 }
+
+.amp-story-editor-carousel-item-wrapper {
+	width: 100%;
+	height: 100%;
+}
+
+.components-draggable__clone .amp-story-editor-carousel-item-wrapper {
+	height: 8px;
+}
+
+.amp-story-editor-carousel-item {
+	position: relative;
+}
+
+.amp-story-editor-carousel-item .components-drop-zone {
+	left: -10px;
+	width: calc(100% + 20px);
+	height: 18px;
+	top: -5px;
+	border: none;
+	border-radius: 0;
+}
+
+.amp-story-editor-carousel-item .components-drop-zone.is-active {
+	background: none;
+}
+
+.amp-story-editor-carousel-item .components-drop-zone.is-close-to-right {
+	border-right: 4px solid #0071a1;
+	left: -8px;
+}
+
+.amp-story-editor-carousel-item .components-drop-zone.is-close-to-left {
+	border-left: 4px solid #0071a1;
+	left: -12px;
+}
+
+.amp-story-editor-carousel-item .components-drop-zone .components-drop-zone__content-text,
+.amp-story-editor-carousel-item .components-drop-zone .components-drop-zone__content-icon,
+.amp-story-editor-carousel-item .components-drop-zone.is-dragging-indicator {
+	display: none;
+}
+
+.indicator-page-number {
+	font-size: 11px;
```

Comment:
That seems rather small.

What about something like this:

```css
.indicator-page-number {
    font-size: 13px;
    display: block;
    margin-top: 10px;
    color: #555d66;
}
```

`#555d66` is also used by Gutenberg in some places. Alternatively, `#40464d` seems to be used as well.